virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc

```diff
@@ -22,3 +22,5 @@ include::modules/virt-cloning-a-datavolume.adoc[leveloffset=+1]
 * xref:../../../virt/virtual_machines/cloning_vms/virt-cloning-vm-disk-into-new-datavolume.adoc#virt-cloning-pvc-of-vm-disk-into-new-datavolume_virt-cloning-vm-disk-into-new-datavolume[Cloning the persistent volume claim of a virtual machine disk into a new data volume]
 
 * xref:../../../virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc#virt-using-preallocation-for-datavolumes[Configure preallocation mode] to improve write performance for data volume operations.
+
+* xref:../../../virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc#virt-customizing-storage-profile_virt-creating-data-volumes[Customizing the storage profile]
```

Comment:
I also wonder if we can improve this `virt-cloning-a-datavolume-using-smart-cloning.adoc` document to cover both `snapshot` and `csi volume clone`? But how to do it so we do not confuse the reader?

@aglitke @bgaydosrh wdyt?

Reply:
Thanks @brybacki. I'll put your comments in soon.

If you and @aglitke think combining is a good idea, we'd have to put it on a future release. The only thing I will say is that with the addition of online snapshots and the needed prereq for QEMU guest agent (a lot of doc right there) the snapshot topic is HUGE (see PR https://github.com/openshift/openshift-docs/pull/36592). But if you think it makes more sense conceptually to combine we could propose it for a tech doc debt story in the future.